readme.md

```diff
@@ -543,6 +544,7 @@
 | [Halfmoon UI](https://www.gethalfmoon.com/)| A responsive and lightweight framework, designed for quickly building beautiful dashboards and product pages. |
 | [Metro 4](https://metroui.org.ua/index.html)| Create your site quickly and effectively with Metro 4. impressive components library built on html, css, javascript. |
 | [css-doodle](https://css-doodle.com/) | A web component for drawing patterns with CSS |
+| [CSS Gradient](https://cssgradient.io/) | CSS Gradient is a little website and free tool that lets you create a gradient background for websites |
```

Comment:
Hey Thanks For the PR! 🎉 

Can you please open separate PR for this Resource, if its not on the list already?

Reply:
[Storyset] -> [UI Graphics] this is anew PR